wp-rocket.php

```diff
@@ -33,6 +33,10 @@
 define( 'WP_ROCKET_PATH',                  realpath( plugin_dir_path( WP_ROCKET_FILE ) ) . '/' );
 define( 'WP_ROCKET_INC_PATH',              realpath( WP_ROCKET_PATH . 'inc/' ) . '/' );
 
+if ( ! defined( 'WP_ROCKET_VIEWS' ) ) {
+	define( 'WP_ROCKET_VIEWS', WP_ROCKET_PATH . 'views' );
```

Comment:
All of the other folder paths have a trailing slash, I think it should be the same for this one imho.